code/datums/brain_damage/hypnosis.dm

```diff
@@ -25,6 +26,8 @@
 		return new type(hypnotic_phrase)
 
 /datum/brain_trauma/hypnosis/on_gain()
+	if(istype(src, /datum/brain_trauma/hypnosis/ipc))
```

Comment:
make a new proc and call that for all the text + flavor, then override it for the IPC trauma